packages/compartment-mapper/src/archive-parsers.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @file Provides a set of default language behaviors (parsers) suitable for
+ * creating archives (zip files with a `compartment-map.json` and a file for
+ * each module) with pre-compiled sources.
+ *
+ * This module entrains a dependency upon the core of Babel.
+ */
```

Comment:
You want [`@packageDocumentation`](https://typedoc.org/tags/packageDocumentation/) or [`@module`](https://typedoc.org/tags/module/) instead of `@file` (which is seemingly unsupported by TypeDoc).

It should appear below the text.

```suggestion
 * Provides a set of default language behaviors (parsers) suitable for
 * creating archives (zip files with a `compartment-map.json` and a file for
 * each module) with pre-compiled sources.
 *
 * This module entrains a dependency upon the core of Babel.
 * 
 * @packageDocumentation
 */
```

Reply:
Elected to leave these as ordinary comments until such a time as we take the trouble to set up typedoc for this package.